playwright.config.ts

```diff
@@ -28,7 +28,7 @@ const config: PlaywrightTestConfig = {
   /* Retry on CI only */
   retries: process.env.CI ? 1 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
```

Comment:
ehhh i m not sure this is going to work out. let's see.

Reply:
well it worked in the CI as you can see with current build

Reply:
The issue we had initially i thought it was with fabric globals.
Text tests were doing some overlapping when run in parallel because of fabricJS globals buffers like the copy-paste text variable or other small config issues.
putting ci to 1 solved the problem, but let's see because then we also found different remediations that were in the playwright api without impacting the number of parallel tests for the other tests

Reply:
> Text tests were doing some overlapping when run in parallel because of fabricJS globals buffers like the copy-paste text variable or other small config issues.

I see that as an issue if you run tests in one test run
like we had before
but now each test is its own isolated page instance and its own dedicated worker
so each test will spawn and import its own fabric global, that is for the browser part

for node part of the tests not sure, each test is own worker, but workers are reused if test passes
but lets see
so far plawyright tests are working